app/ts/components/pages/PersonalSign.tsx

```diff
@@ -155,17 +162,48 @@ function SignRequest({ personalSignRequestData, renameAddressCallBack }: SignReq
 }
 
 
+const CellElement = (param: { text: ComponentChildren }) => {
+	return <div class = 'log-cell' style = 'justify-content: right;'> <p class = 'paragraph' style = 'color: var(--subtitle-text-color)'> { param.text }</p></div>
+}
+
+export function Permit2ExtraDetails({ permit2 }: { permit2: PersonalSignRequestDataPermit2 }) {
+	return <>
+		<CellElement text = 'Chain: '/>
+	<	CellElement text = { getChainName(BigInt(permit2.message.domain.chainId)) }/>
+		<CellElement text = 'Nonce: '/>
+		<CellElement text = { permit2.message.message.details.nonce.toString(10) }/>
+		<CellElement text = 'Signature expires  in:'/>
+		<CellElement text = { <SomeTimeAgo priorTimestamp = { new Date(Number(permit2.message.message.sigDeadline) * 1000) } countBackwards = { true }/> }/>
+		<CellElement text = 'Spender can spend for:'/>
+		<CellElement text = { <>
+			<SomeTimeAgo priorTimestamp = { new Date(Number(permit2.message.message.details.expiration) * 1000) } countBackwards = { true }/>
+			{` (until ${ new Date(Number(permit2.message.message.details.expiration) * 1000).toISOString().split('T')[0] })`}
```

Comment:
date localization maybe? `new Date().toLocaleDateString("en-US")`

Reply:
I have a pretty strong preference for ISO8601 dates rather than localized dates (because most date localizations are incredibly stupid and the world needs to move towards a global standard).  The US localization in particular is literally the worst of them all.  😆